task/expression/template_resolver.go

```diff
@@ -0,0 +1,47 @@
+package expression
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/drone/go-task/task/common"
+)
+
+const (
+	RESOLVER_DELIM_START = "#{{"
```

Comment:
use #{{  instead of <- 
because <- is very commonly used in script when people trying to write an left arrow. This may cause many unnecessary string parsing